### ISSUE TITLE
Plugin suffix for plugins

### ISSUE THREAD

**Issue by dereuromark** [spryker/architecture-sniffer#54]
this was reported:

"and what about `naming` sniffs? that `…PluginInterface` should be implemented by `…Plugin`
otherwise this looks suspicious"

```php
    protected function getQuoteStorageStrategyPlugins()
    {
        return [
            new SessionQuoteStorageStrategyPlugin(),
            new DatabaseQuoteStorageStrategy(),
        ];
    }
```

=> always Plugin suffix

see https://github.com/spryker/persistent-cart/blob/master/src/Spryker/Client/PersistentCart/Plugin/DatabaseQuoteStorageStrategy.php
